webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/annotation/BratAnnotationEditor.java

```diff
@@ -172,6 +172,9 @@ else if (!request.getParameterValue(PARAM_ID).isEmpty()) {
                 else {
                     paramId = VID.parseOptional(request.getParameterValue(PARAM_ARC_ID).toString());
                 }
+                if (paramId.isSynthetic()) {
```

Comment:
Can this not happen in the "isSynthetic" branch below? I believe that here we only need to check if the VID contains an extension ID so we can route events to the extension. The receiving extension then needs to see what it can get from the payload of the VID.